app/src/main/java/de/westnordost/streetcomplete/osm/Surface.kt

```diff
@@ -1,12 +1,36 @@
 package de.westnordost.streetcomplete.osm
 
-val ANYTHING_UNPAVED = setOf(
+val SOFT_SURFACES = setOf("ground", "earth", "dirt", "grass", "sand", "mud", "ice", "salt", "snow", "woodchips")
+
+val ANYTHING_UNPAVED = SOFT_SURFACES + setOf(
     "unpaved", "compacted", "gravel", "fine_gravel", "pebblestone", "grass_paver",
-    "ground", "earth", "dirt", "grass", "sand", "mud", "ice", "salt", "snow", "woodchips"
 )
 
-val ANYTHING_PAVED = setOf(
+val ANYTHING_FULLY_PAVED = setOf(
     "paved", "asphalt", "cobblestone", "cobblestone:flattened", "sett",
-    "concrete", "concrete:lanes", "concrete:plates", "paving_stones",
+    "concrete", "concrete:plates", "paving_stones",
     "metal", "wood", "unhewn_cobblestone"
 )
+
+val ANYTHING_PAVED = ANYTHING_FULLY_PAVED + setOf(
+    "concrete:lanes"
+)
+
+fun isSurfaceAndTractypeMismatching(surface: String, tracktype: String): Boolean {
```

Comment:
Typo: missing `k`
Also `track type` as camelCase should be `trackType` in parameter name and function name (and other places).

Partly fixed in my suggestion in next comment.

Reply:
> camelCase should be trackType in parameter name and function name (and other places) – even though tracktype is the tag used in OSM.

not entirely sure, I treat "tracktype" as a word/code. Is "track type" even a thing in actual English? I thought that it is osmism not matching actual English terms.